### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by Andrew1995bono**
A Footer section was created.

### CODE REVIEW

File: src/app/components/all-products-components/all-products-content/product-card/product-card.component.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
+.product-item {
+  display: flex;
+  justify-content: space-between;
+  width: 869px;
+  height: 280px;
+  /*margin-top: 81px;*/
```
Comment (SStegniy) [inline src/app/components/all-products-components/all-products-content/product-card/product-card.component.css:6]: Remove comment

File: src/app/components/homepage-components/navbar/select-field/select-field.component.css [hunk 2]
```diff
@@ -2,11 +2,8 @@
   display: flex;
   align-items: center;
   height: 55px;
-  top: 100px;
-  left: 17px;
   background: #F9F9F9;
   margin-left: 17px;
-  /*!*width: 1170px;*! Use this property to correct styling width */
```
Comment (SStegniy) [inline src/app/components/homepage-components/navbar/select-field/select-field.component.css:9]: remove comment line

File: src/styles.css [hunk 3]
```diff
@@ -1,11 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
 
 html {
-  box-sizing: border-box;
   margin: 0 auto;
   padding: 16px 45px 78px;
   max-width: 1260px;
-  height: 100vh;
+  /*height: 100vh;*/
```
Comment (SStegniy) [inline src/styles.css:7]: please remove

**Review by SStegniy — APPROVED** [review]

File: src/app/components/all-products-components/all-products-content/product-card/product-card.component.css (hunk 1 above)
Comment (Andrew1995bono) [reply src/app/components/all-products-components/all-products-content/product-card/product-card.component.css:6]: +fixed

File: src/styles.css (hunk 3 above)
Comment (Andrew1995bono) [reply src/styles.css:7]: fixed

File: src/app/components/homepage-components/navbar/select-field/select-field.component.css (hunk 2 above)
Comment (Andrew1995bono) [reply src/app/components/homepage-components/navbar/select-field/select-field.component.css:9]: fixed

**Pull request merged**